src/nvim/ui.coffee

```diff
@@ -206,8 +206,13 @@ class UI extends EventEmitter
   nv_cursor_goto: (row, col) ->
     @cursor_row = row
     @cursor_col = col
-  nv_cursor_off: -> @cursor.style.display = 'none'
-  nv_cursor_on: -> @cursor.style.display = 'block'
+
```

Comment:
The handlers are ordered in alphabetical order. And the `nv_cursor_*` functions are grouped together without blank lines in between.